lib/charms/loki_k8s/v0/log_proxy.py

```diff
@@ -586,26 +599,48 @@ def _scrape_configs(self) -> dict:
             A dict representing the `scrape_configs` section.
         """
         # TODO: use the JujuTopology object
-        config = {
-            "targets": ["localhost"],
-            "labels": {
-                "job": "juju_{}_{}_{}".format(
-                    self._charm.model.name,
-                    self._charm.model.uuid,
-                    self._charm.model.app.name,
-                ),
-                "__path__": "",
-            },
+        job_name = "juju_{}_{}_{}".format(
+            self._charm.model.name, self._charm.model.uuid, self._charm.model.app.name
+        )
+        common_labels = {
+            "juju_application": self._charm.app.name,
+            "juju_model": self._charm.model.name,
+            "juju_model_uuid": self._charm.model.uuid,
+            "juju_charm": self._charm.meta.name,
         }
 
-        return {
-            "scrape_configs": [
-                {
-                    "job_name": "system",
-                    "static_configs": self._generate_static_configs(config),
-                }
-            ]
+        scrape_configs = []
+
+        # Files config
+        labels = deepcopy(common_labels)
+        labels.update(
+            {
+                "job": job_name,
+                "__path__": "",
+            }
+        )
+        config = {"targets": ["localhost"], "labels": labels}
+        scrape_config = {
+            "job_name": "system",
+            "static_configs": self._generate_static_configs(config),
```

Comment:
If the LogProxyConsumer is instantiated without a list of files, the `static_configs` will be empty... the generated config file is going to work in this case?

Reply:
Gotta put a test on it :muscle:

Reply:
This would be an integration test so we should write it after the lib merge. I'm pretty sure it worked in my testing.

Reply:
It will return an empty list, which Prometheus is ok parsing